components/table/test/table.vdiff.js

```diff
@@ -424,181 +425,183 @@ describe('table', () => {
 				});
 			});
 
-			describe('sticky', () => {
-				const inputRow = html`
-					<tr class="down">
-						<td>Cell 2-A</td>
-						<td><d2l-input-text label="label" label-hidden value="Cell 2-B" input-width="100px"></d2l-input-text></td>
-						<td>Cell 2-C</td>
-					</tr>
-				`;
-				[
-					{
-						name: 'one-row-thead',
-						template: html`
-								<thead>${createHeaderRow({ trClass: 'top' })}</thead>
+			[true, false].forEach((hasStickyHeadersScrollWrapper) => {
+				describe(`sticky${hasStickyHeadersScrollWrapper ? '-scroll-wrapper' : ''}`, () => {
```

Comment:
So this reproduces all the sticky tests in two forms now, with and without the scroll-wrapper. I'm not sure whether we need all of these, or whether we want more targeted vdiffs for the specific behaviour of `sticky-headers-scroll-wrapper` - but I'll leave that for Gaudi to decide.

Reply:
hah yeah, there were already quite lot of sticky cases with _default vs light_ and _ltr vs rtl_.  I was concerned that any one those might behave uniquely, so I added the _scroll-wrapper_ scenario for that entire suite. If we're thinking that is overkill I can certainly scale it back a bit 😄

Reply:
Yeah that was my first reaction. The visual-diffs certainly aren't "free" so we don't necessarily always capture every permutation of every property, state, etc. If they are worthwhile, then it's fine, but it's also a lot. 

It would be nice to have a callout in the ones that are showing broken behaviour.

Reply:
Thank you both for the great feedback!

I do think there will be value in having both _ltr and rtl,_ but _default vs light_ don't seem as likely to make a difference 🤔  I'll reflect further and see what I can cut down on. 

Also really like the idea of calling out the ones that seem busted. I'll update this thread when I've got something that takes these considerations into account ...